examples/nidaqmx_analog_input/NIDAQmxAnalogInput.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.NIDAQmxAnalogInput_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "NI-DAQmx Analog Input Example",
+        "ni/service.collection": "Examples",
```

Comment:
Change all collections to NI.Examples

This change was suggested [this related PR](https://dev.azure.com/ni/DevCentral/_git/ASW/pullrequest/510451).